.gitignore

```diff
@@ -1,7 +1,17 @@
+# Ignore NPM generated files
+node_modules/
+npm-debug.log
+
+# Ignore OS generated files
+.DS_Store*
+Thumbs.db
+
```

Comment:
please do not add these OS X specific things. see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore